test/helpers/shallowRenderHelper.js

```diff
@@ -6,7 +6,8 @@
  * @author somonsmith
  */
 import React from 'react'
-import TestUtils from 'react-dom/test-utils'
+import ShallowRenderer from 'react-test-renderer/shallow'
```

Comment:
I didn't know about [this shallow rendering testing strategy](https://reactjs.org/docs/shallow-renderer.html). Nice!